### DISCUSSION TITLE
Radeon Pro Render for Maya 2023 + feature requests

### DISCUSSION THREAD

**Discussion by budsyakorn** [GPUOpen-LibrariesAndSDKs/RadeonProRenderMayaPlugin#292] [Ideas]
I want Radeon Pro render to compatible with the latest Maya 2023 that was released a week ago. It has so many cool features out here.
[https://help.autodesk.com/view/MAYAUL/2023/ENU/?guid=GUID-3B38E9FB-CD86-41DF-8104-109A13815B96](url)

My Features requests
- I want this "override shadow color" and the other features that applied to Blender 3.3.16, into Maya. This will be a very helpful feature to create a different style approach. The current version of this renderer lacks tons of these.
- Fully Bifrost support - Bifrost is a great tool to create water and particle animation with ease. The current version can't render Bifrost animation without converting to mesh. It's better to make this renderer to fully support this Bifrost engine - it will save lots of time as well.

**Comment by bsavery** [comment]
Hi There!  We just got the 2023 SDK and taking a look. 

Adding the shader color setting is coming soon.  
Bifrost support is much bigger ask though!

**Reply by yucelyilmazvfx** [reply]
Is it ready? Still waiting for Maya 2023. Thanks.
